sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -7,7 +7,7 @@
 # pylint: disable=super-init-not-called, too-many-lines
```

Comment:
Just putting this at the top of `_models.py`

I am unsure how we want to approach this. I originally put it here because it made sense that Blob's Audience is only known by Blob, but then I am not sure how we are going to type hint it if we pass the actual Audience object around. An alternative is that we can process out the value and once it hits base_client, `audience` will be a string.